locales/zh_CN/LC_MESSAGES/export_results.po

```diff
@@ -131,65 +161,79 @@ msgstr ""
+"by **the number of channels in the input tensor**, i.e., the length of the "
+"third dimension (``tensor.shape[2]``)."
 msgstr ""
+"同时，得到的图像类型（灰度、RGB 或 RGBA）由 **输入张量的通道数** 决定。即，第"
+"三维的长度( ``tensor.shape[2]`` )。"
 
 #: ../../export_results.rst:73
 msgid ""
-"In other words, a tensor that has shape ``(w, h)`` or ``(w, h, 1)`` will "
-"be exported as a grayscale image."
-msgstr ""
+"In other words, a tensor that has shape ``(w, h)`` or ``(w, h, 1)`` will be "
+"exported as a grayscale image."
+msgstr "换言之，形状是 ``(w, h)`` 或 ``(w, h, 1)`` 的张量会被导出成灰度图。"
 
 #: ../../export_results.rst:74
 msgid ""
-"If you want to export ``RGB`` or ``RGBA`` images instead, the input "
-"tensor should have a shape ``(w, h, 3)`` or ``(w, h, 4)`` respectively."
+"If you want to export ``RGB`` or ``RGBA`` images instead, the input tensor "
+"should have a shape ``(w, h, 3)`` or ``(w, h, 4)`` respectively."
 msgstr ""
+"如果你想导出 ``RGB`` 或 ``RGBA`` 的图像，输入的张量形状应该分别是 ``(w, h, "
+"3)`` 或 ``(w, h, 4)``。"
 
 #: ../../export_results.rst:78
 msgid ""
-"All Taichi tensors have their own data types, such as ``ti.u8`` and "
-"``ti.f32``. Different data types can lead to different behaviors of "
-"``ti.imwrite``. Please check out :ref:`gui` for more details."
+"All Taichi tensors have their own data types, such as ``ti.u8`` and ``ti."
+"f32``. Different data types can lead to different behaviors of ``ti."
+"imwrite``. Please check out :ref:`gui` for more details."
 msgstr ""
+"Taichi 中所有的张量都有自己的数据类型，比如 ``ti.u8`` 和 ``ti.f32``。不同的数"
+"据类型会导致 ``ti.imwrite`` 产生不同的输出。请参阅 :ref:`gui` 了解更多细节。"
 
 #: ../../export_results.rst:80
 msgid ""
-"Taichi offers other helper functions that read and show images in "
-"addition to ``ti.imwrite``. They are also demonstrated in :ref:`gui`."
+"Taichi offers other helper functions that read and show images in addition "
+"to ``ti.imwrite``. They are also demonstrated in :ref:`gui`."
 msgstr ""
+"除了 ``ti.imwrite`` 之外，Taichi 还提供了其他读取和显示图像的辅助函数。在 :"
+"ref:`gui` 中也会有它们的示例。"
 
 #: ../../export_results.rst:83
 msgid "Export videos"
-msgstr ""
+msgstr "导出视频"
 
 #: ../../export_results.rst:87
 msgid ""
-"The video export utilities of Taichi depend on ``ffmpeg``. If ``ffmpeg`` "
-"is not installed on your machine, please follow the installation "
-"instructions of ``ffmpeg`` at the end of this page."
+"The video export utilities of Taichi depend on ``ffmpeg``. If ``ffmpeg`` is "
+"not installed on your machine, please follow the installation instructions "
+"of ``ffmpeg`` at the end of this page."
 msgstr ""
+"Taichi 的视频导出工具依赖于 ``ffmpeg``。如果你的机器上还没有安装 ``ffmpeg``，"
+"请按照本节末尾的 ``ffmpeg`` 安装说明进行操作。"
 
 #: ../../export_results.rst:89
 msgid ""
-"``ti.VideoManger`` can help you export results in ``mp4`` or ``gif`` "
-"format. For example,"
-msgstr ""
+"``ti.VideoManger`` can help you export results in ``mp4`` or ``gif`` format. "
+"For example,"
+msgstr "``ti.VideoManger`` 可以帮助你导出 mp4 或 gif 格式的结果。例如,"
```

Comment:
@Rullec typo again? Should be `Manager` here, could you fix it in main repo later?

Reply:
Thanks for point it out, I will fix it tonight.

Is there still any other typo? The `AssertationError` we mentioned above is not a typo, it matches the behavior of current Taichi. Though whether it is suitable is another question.

Reply:
Ok, but you said `ValueError` above:
> .. note:: The dimensionality of tensor shapes should **be consistent** with that of the offset. Otherwise, a ``ValueError`` will be raised.

So the typo may be the `ValueError` here?